src/app/sponsors/sponsors.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="text-center" [@fade]="'in'">
+    <h2 class="text-5xl text-lightCyan">Thank You to Our Sponsors!</h2>
```

Comment:
Would you mind changing this to sentence casing? We're using sentence casing in all the slides (including the ones in Google Slides which I know you don't see here as a reference 😄 : ).

And also the casing for "Partner Sponsors" and "Contributor Sponsors". TY!